### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by ZH-Lee** [YunYang1994/tensorflow-yolov3#205]
In giou, why using pred_bbox and anchor_box, instead of  true_bbox and pred_bbox?
